test/stableX_large.js

```diff
@@ -0,0 +1,183 @@
+const StablecoinConverter = artifacts.require("StablecoinConverter")
+const MockContract = artifacts.require("MockContract")
+
+const truffleAssert = require("truffle-assertions")
+
+const {
+  waitForNSeconds,
+  sendTxAndGetReturnValue
+} = require("./utilities.js")
+
+const feeDenominator = 1000 // fee is (1 / feeDenominator)
+
+function feeSubtracted(x) {
+  return Math.floor(x * (feeDenominator - 1) / feeDenominator)
+}
+
+function feeAdded(x) {
+  return Math.floor(x * (feeDenominator) / (feeDenominator - 1))
+}
```

Comment:
Maybe these functions can be required from a more general place.

Reply:
Yes, I can move them into utilities if you like.

Reply:
Wait.. this Branch is never going to be merged....